_posts/articles/2020-11-06-ten-tips-maintaining-long-term-docs-like-code.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: post
+title: Ten tips for maintaining a long-term relationship with docs like code
+excerpt: "Learn about one team's journey to keep a long-term relationship with their docs-like-code system, TechDocs, which is open source and available to all."
+last_modified_at: Fri Nov  6 20:47:45 CST 2020
+categories: articles
+author: gary_niemen
+tags: [cicd, backstage, spotify, github, git, portal, internal, documentation, developer, docs]
+image:
+  path: /images/spotify/soundboard-han.ailes.jpg
+  caption: "[Flickr han.ailes](https://flic.kr/p/oVPTVm)"
+comments: false
+share: true
+---
+
+<!-- By Gary Niemen, Product Manager for TechDocs, Spotify’s docs-like-code solution -->
+
+I remember about two years ago sitting here watching a Write the Docs video. Well, not precisely “here” because we sat in offices in those days and in 2020 I'm in my home office working for Spotify. If you haven't heard of it, Spotify is a digital music, podcast, and video streaming service providing access to content from artists around the world. I was watching, wide-eyed and a bit confused, a video of [Riona MacNamara’s talk from Write the Docs 2015](https://youtu.be/EnB8GtPuauw). In the talk, Riona described how she and another technical writer used a docs-like-code approach to change Google engineering culture around technical documentation.
+
```

Comment:
I split the lead into three sentences and I added a description of what Spotify is with, "If you haven't heard of it, Spotify is a digital music, podcast, and video streaming service providing access to content from artists around the world." I paraphrased from a support article so hopefully your review folks will still approve. :)

Reply:
Good catch. I am normally good on giving people the right context - but I had totally forgotten that piece. I have read through this paragraph a few times. I am not sure it flows 100% because I was working for Spotify back then as well and it reads like I have recently joined. But perhaps I am just being pedantic. I can't think of a better way to express it. So perhaps it is fine. Read through yourself just to check - and then I trust your judgement.Everything else is fine. I sent the pic of me on Slack. Thanks for the great collaboration.